### DISCUSSION TITLE
provide global hooks?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#13196] [Feature Requests]
For example, global global hooks before entering the router, and hooks when returning api data

Similar to the vue router method

https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards

beforeEach and afterEach

**Comment by rijkvanzanten** [comment]
Have you seen the currently available hooks? 🙂

https://docs.directus.io/extensions/hooks/

**Reply by ghost** [reply]
sorry 
thinks
